confluent_platform/datadog_checks/confluent_platform/data/metrics.yaml

```diff
@@ -0,0 +1,1276 @@
+        # Fetch request rate.
+        - kafka.server:type=BrokerTopicMetrics,name=TotalFetchRequestsPerSec
+        # Produce request rate.
+        - kafka.server:type=BrokerTopicMetrics,name=TotalProduceRequestsPerSec
+      attribute:
+        Count:
+          alias: $domain.topic.$name.rate
+          metric_type: rate
+
+  # Kafka Broker Topic Metrics Regex (Yammer Metric Meter)
+  - include:
+      domain: kafka.server
+      bean:
+        # Rate at which the pool of in-sync replicas (ISRs) expands.
+        - kafka.server:type=ReplicaManager,name=IsrExpandsPerSec
+        # Rate at which the pool of in-sync replicas (ISRs) shrinks.
+        - kafka.server:type=ReplicaManager,name=IsrShrinksPerSec
+      attribute:
+        Count:
+          alias: $domain.$type.$name.rate
+          metric_type: rate
+  - include:
+      domain: kafka.server
+      bean:
+        # Average fraction of time the request handler threads are idle. Values are between 0 (all resources are used) and 1 (all resources are available)
+        - kafka.server:type=KafkaRequestHandlerPool,name=RequestHandlerAvgIdlePercent
+      attribute:
+        Count:
+          alias: $domain.request_handler_pool.avg_idle_percent.rate
+          metric_type: rate
+
+  # Kafka Broker ZooKeeper Metrics (Yammer Metric Meter)
+  - include:
+      domain: kafka.server
+      bean:
+        # ZooKeeper client is currently disconnected from the ensemble. The client lost its previous connection to a server and it is currently trying to reconnect. The session is not necessarily expired.
+        - kafka.server:type=SessionExpireListener,name=ZooKeeperDisconnectsPerSec
+        # The ZooKeeper session has expired. When a session expires, we can have leader changes and even a new controller.
+        - kafka.server:type=SessionExpireListener,name=ZooKeeperExpiresPerSec
+        # ZooKeeper client is connected to the ensemble and ready to execute operations.
+        - kafka.server:type=SessionExpireListener,name=ZooKeeperSyncConnectsPerSec
+        # An attempt to connect to the ensemble failed because the client has not provided correct credentials.
+        - kafka.server:type=SessionExpireListener,name=ZooKeeperAuthFailuresPerSec
+        # The server the client is connected to is currently LOOKING, which means that it is neither FOLLOWING nor LEADING.
+        - kafka.server:type=SessionExpireListener,name=ZooKeeperReadOnlyConnectsPerSec
+        # Client has successfully authenticated.
+        - kafka.server:type=SessionExpireListener,name=ZooKeeperSaslAuthenticationsPerSec
+      attribute:
+        Count:
+          alias: $domain.session.$name.rate
+          metric_type: rate
+
+  # Kafka Broker Request Metrics (Yammer Metric Meter)
+  - include:
+      domain: kafka.network
+      bean_regex:
+        # Request rate.
+        - kafka.network:type=RequestMetrics,name=RequestsPerSec,request=(Produce|FetchConsumer|FetchFollower).*
```

Comment:
Does that mean that three different bean will be submitted under the same metric name? Should we not tag by the request here?

Reply:
Answered here https://github.com/DataDog/integrations-core/pull/5733#discussion_r392903038

Reply:
By default all key/value pairs from the bean name are used as tags.